.github/workflows/ansible-release.yml

```diff
@@ -49,10 +47,39 @@ jobs:
       - name: Install dependencies
         working-directory: antsibull
         run: |
+          python3 -m pip install packaging
           python3 -m pip install ansible-core
           python3 -m pip install antsibull
```

Comment:
TODO in a separate PR: merge these into one pip install command instead of three

Reply:
I was wondering whether the separate installs were intentional for some reason, so I added a third. But yeah, let's collapse them (in a separate PR).